src/test/java/seedu/guest/testutil/EditGuestDescriptorBuilder.java

```diff
@@ -85,7 +85,7 @@ public EditPersonDescriptorBuilder withAddress(String address) {
      * Parses the {@code tags} into a {@code Set<Tag>} and set it to the {@code EditPersonDescriptor}
```

Comment:
Should EditPersonDescriptor be refactored to EditGuestDescriptor? (occurs multiple times in this file)